code/src/screens/Auth/ForgotBarcode.js

```diff
@@ -129,14 +130,9 @@ export const ForgotBarcode = (props) => {
                          </ModalHeader>
                          <ModalBody>
                               {showResults && !results.success ? (
-                                   <>
-                                        <Text color={textColor}>{stripHTML(results.message)}</Text>
-                                        <Button bgColor={theme['colors']['primary']['500']} onPress={resetWindow}>
-                                             <ButtonText color={theme['colors']['primary']['500-text']}>{getTermFromDictionary('en', 'try_again')}</ButtonText>
-                                        </Button>
-                                   </>
+                                   <Text color={textColor}>{stripHTML(results.message || 'An unknown error occurred. Please contact the library for assistance.')}</Text>
```

Comment:
text should be added into defaults.json and called with getTermFromDictionary() so its translatable